### PR TITLE
[QC-419] Integrate ObjectManager with PostProcessing

### PR DESCRIPTION
**Pull request by knopers8**

### CODE REVIEW

**Comment by knopers8** [comment]
BTW, service discovery should work gratuit

**Comment by knopers8** [comment]
@Barthelemy testCcdbDatabase has randomly failed again.
[testccdbdatabase.log](https://github.com/AliceO2Group/QualityControl/files/5303542/testccdbdatabase.log)

**Comment by Barthelemy** [comment]
thank you for saving the logs

File: Framework/include/QualityControl/PostProcessingRunner.h [hunk 1]
```diff
@@ -73,13 +88,19 @@ class PostProcessingRunner
 
   std::unique_ptr<PostProcessingInterface> mTask;
   framework::ServiceRegistry mServices;
+  std::shared_ptr<o2::quality_control::core::ObjectsManager> mObjectManager;
+  // TODO in a longer run, we should store from/to in the MonitorObject itself and use them.
```
Comment (Barthelemy) [inline Framework/include/QualityControl/PostProcessingRunner.h:92]: agreed

**Comment by Barthelemy** [comment]
very good, I like the callback and how you can reuse the ObjectsManager (and thus have the ServiceDiscovery for free!).

**Review by Barthelemy — APPROVED** [review]

**Comment by Barthelemy** [comment]
@awegrzyn what is this second travis ? any idea ?

**Comment by awegrzyn** [comment]
idk, there should be only 1

**Pull request merged**